spec/plans/results.fmf

```diff
@@ -27,7 +27,7 @@ description: |
          path: /
 
        # String, outcome of the test execution.
-       result: "pass"|"fail"|"info"|"warn"|"error"
+       result: "pass"|"fail"|"info"|"warn"|"error"|"not executed"
```

Comment:
Would `skipped` work for you? No space, and seems to be aligned with frameworks like pytest or junit seem to use "skip" or "skipped".